docs/topics/tour/kotlin-tour-null-safety.md

```diff
@@ -87,7 +87,7 @@ fun main() {
 ## Use safe calls
 
 To safely access properties of an object that might contain a `null` value, use the safe call operator `?.`. The safe call
-operator returns `null` if the object's property is `null`. This is useful if you want to avoid the presence of `null`
+operator returns `null` if the object is `null`. This is useful if you want to avoid the presence of `null`
```

Comment:
JFYI
I think the important part is `the object` itself is `null`,
but not about `the object's property`

Reply:
I think the `String.length` property is never `null`
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-string/length.html

the point here is to check whether the object `maybeString` itself is `null` or not
if the object itself is `null`, the `safe call operator` returns `null`
or, if the object itself is not `null`,  the `safe call operator` returns it's property value as-is

Reply:
I see your point 🤔 - very well put! 👍 

I propose including both - the text itself doesn't just cover the example following it especially that the note talks about chaining safe calls. 

"The safe call operator returns `null` if either the object or one of its accessed properties is `null`."

This way we can make it all-encompassing.

Reply:
thank you, I have fixed it as your suggestion
please check it again.